app/webpack/shared/components/text_editor.jsx

```diff
@@ -7,12 +7,21 @@ class TextEditor extends React.Component {
   constructor( props, context ) {
     super( props, context );
     this.textarea = React.createRef();
+    this.boldButton = React.createRef();
+    this.italicButton = React.createRef();
+    this.linkButton = React.createRef();
     this.state = {
       textareaChars: 0,
       preview: false
     };
   }
 
+  componentDidMount( ) {
+    Mousetrap( this.textarea.current ).bind( 'mod+b', ( ) => { this.boldButton.button.current.click( ); } );
```

Comment:
While this is working, Mousetrap should be imported or declared as a global in an eslint comment. Also, double-quotes only, please. React-specific eslint rules are at https://github.com/inaturalist/inaturalist/blob/master/app/webpack/.eslintrc